docs/docs-content/clusters/edge/local-ui/host-management/access-console.md

```diff
@@ -69,14 +69,54 @@ A successful sign-out takes you back to the login page. You need to enter your c
 
 ## Change User Password
 
+You can change the password of an OS user through Local UI, through the terminal, or through the Palette API.
+
 ### Prerequisites
 
 - Network access to the Edge host's port where Local UI is exposed. Default is port 5080.
 
 - Credentials to log in to Local UI. Any OS user can be used to log into Local UI.
 
+- Depending on how you want to change the password, additional prerequisites are required.
+
+  <Tabs>
+
+  <TabItem value="Local UI">
+
+  - Through Local UI: The `stylus.disablePasswordUpdate` parameter is not set to `true` in the installer configuration
+    user data. The default value of the parameter is `false`. For more information, refer to
+    [Installer Reference](../../edge-configuration/installer-reference.md#local-ui).
+
+  </TabItem>
+
+  <TabItem value="Terminal">
+
+  - None.
+
+  </TabItem>
+
+  <TabItem value="API">
+
+  - Through the API: [curl](https://curl.se/docs/install.html) is installed on the machine you plan to use to change the
```

Comment:
curl or similar HTTP tool such as wget. It's good to have examples, but let's make it a bit more clear that `curl` is not required.